src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -1308,7 +1308,10 @@ static void ipa_s2n_get_list_next(struct tevent_req *subreq)
     ret = sysdb_attrs_get_string(state->attrs->sysdb_attrs, SYSDB_SID_STR,
                                  &sid_str);
     if (ret != EOK) {
-        DEBUG(SSSDBG_OP_FAILURE, "sysdb_attrs_get_string failed.\n");
+        DEBUG(SSSDBG_CRIT_FAILURE,
+              "Group [%s] has no SID, please check the "
```

Comment:
Are we sure that it cannot be user but only group?

It is not clear for me from this function. Otherwise LGTM.